array_api_compat/sparse/__init__.py

```diff
@@ -0,0 +1,5 @@
+from sparse import * # noqa: F403
+from ..common._aliases import * # noqa: F403
```

Comment:
This is not right. The aliases in common._aliases need to be wrapped in sparse/_aliases.py to work. See https://data-apis.org/array-api-compat/dev/implementation-notes.html

Furthermore, you should only wrap the functions that actually need wrapping. 

If you prefer, you can put all the compatibility shims for sparse in sparse itself, and keep the support here just limited to the helper functions. This is how it currently works for JAX. Since you're also the maintainer of sparse, I would suggest this approach.

Reply:
Thanks for the insight -- fixed in https://github.com/data-apis/array-api-compat/pull/134/commits/a4966e7f64309581e3003985f72bef6ce2787a8b.